kumascript/src/api/web.ts

```diff
@@ -152,9 +155,11 @@ const web = {
           )}"`;
         }
         content ??= enUSPage.short_title ?? enUSPage.title;
+        const title = ONLY_AVAILABLE_IN_ENGLISH(this.env.locale);
+
         return (
           '<a class="only-in-en-us" ' +
-          'title="Currently only available in English (US)" ' +
+          `title="${title}" ` +
```

Comment:
Do we need to escape `"` chars in `title`? `ONLY_AVAILABLE_IN_ENGLISH` is defined out of context, a rogue `"` could very possibly work its way in there. Perhaps `localeString` should take a second argument for characters to escape, if we see this being used more often going forward?

Reply:
Fixed in [8f0236c](https://github.com/mdn/yari/pull/11011/commits/8f0236c42be98ec078e8f45a9b0f617477f65490).